src/popup/modules/UserInterface.js

```diff
@@ -447,20 +480,23 @@ function triggerFileSave(file, filename, requestDownloadPermissions) {
  */
 function menuClicked(event) {
     const requestDownloadPermissions = browser.permissions.request(DOWNLOAD_PERMISSION);
-
+    var filename = generateFilename();
```

Comment:
Don't use `var`, that's outdated. Again, eslint should tell you to use const or let.